scripts/motdgen

```diff
@@ -13,6 +13,15 @@ source /usr/lib/os-release
 mkdir -p /run/coreos
 echo -e "Core\033[38;5;206mO\033[38;5;45mS\033[39m ${GROUP} (${VERSION})" > /run/coreos/motd
 
+mkdir -p /etc/motd.d
+# run any scripts in this directory and add output to motd
```

Comment:
Hm, I would rather this just be static files. At some point I'd like to set up a new scheme for updating the "failed units" list that doesn't require this script to run once a minute. Either by somehow triggering it on events or by replacing it with a small daemon that listens to systemd via dbus. In either case it will be easy to watch /etc/motd.d via inotify for changes and we don't risk scripts depending on the current once a minute behavior.

How about something like this:
```sh
cat /etc/motd.d/*.conf 2>/dev/null >> /run/coreos/motd || true
```

If we add motd.d I vote not bothering with motd.tail